### PR TITLE
Ofstests

### PR DESCRIPTION
**Pull request by patrick-tripp**
Fixes for running models.

### CODE REVIEW

**Comment by patrick-tripp** [comment]
There are more minor commits coming. Unable to set the PR to draft after opening it.

**Comment by patrick-tripp** [comment]
Ready to merge this into main.

File: cloudflow/job/jobs/liveocean.fcst [hunk 1]
```diff
@@ -1,15 +1,15 @@
 {
   "JOBTYPE"   : "romsforecast",
   "OFS"       : "liveocean",
-  "CDATE"     : "today",
+  "CDATE"     : "20170601",
   "HH"        : "00",
-  "COMROT"    : "/com",
+  "COMROT"    : "/com/patrick.tripp",
```
Comment (jonmjoyce) [inline cloudflow/job/jobs/liveocean.fcst:6]: Remove personal directory from this commit

File: cloudflow/workflows/workflow_main.py [hunk 2]
```diff
@@ -32,7 +32,7 @@
 
 # This is used for obtaining liveocean forcing data
 # LiveOcean users need to obtain credentials from UW
-sshuser = 'username@boiler.ocean.washington.edu'
+sshuser = 'username@apogee.ocean.washington.edu'
```
Comment (jonmjoyce) [inline cloudflow/workflows/workflow_main.py:35]: Should probably have an email/username credentials section in the config for cases like this

**Review by jonmjoyce — COMMENTED** [review]
Code looks good, long-term probably don't want that email in workflow_main.py is all

File: cloudflow/job/jobs/liveocean.fcst (hunk 1 above)
Comment (patrick-tripp) [reply cloudflow/job/jobs/liveocean.fcst:6]: This is an example job file. In the future we might want to set it automatically.

File: cloudflow/workflows/workflow_main.py (hunk 2 above)
Comment (patrick-tripp) [reply cloudflow/workflows/workflow_main.py:35]: I agree, this should be set somewhere else. It is just a generic placeholder until then. Thanks.

**Review by jonmjoyce — APPROVED** [review]

**Pull request merged**